### ISSUE TITLE
Pull Requests are being merged without explicit user confirmation

### ISSUE THREAD

**Issue by adamczykba** [servicecatalog/oscm-rest-api#31]
**Describe the bug**
We've noticed that sometimes pull requests are being merged automatically without confirmation from the user.

**To Reproduce**
There are no specific steps yet. Let's observe the repository and try to find something that's common for all of the cases.

**Expected behavior**
Pull Requests are merged only after all checks have passed and user responsible for merging particular PR, explicitly triggered the merging action.

**Observed behavior**
Pull Requests are being merged without user explicit approval and even no checks have passed.

**Screenshots**
![obraz](https://user-images.githubusercontent.com/47355402/56351676-f2ba5b00-61cd-11e9-85a6-65400a7b584f.png)
![obraz](https://user-images.githubusercontent.com/47355402/56352018-a4598c00-61ce-11e9-915f-7caacb8bfda2.png)
![obraz](https://user-images.githubusercontent.com/47355402/56352024-a885a980-61ce-11e9-95c1-5c7e6094ce4c.png)

**Additional context**
- the issue does not appear on main oscm repository
- for now branch protection settings are the same as on the main oscm repo, but checks are enforced on administrative users also

**Comment by adamczykba** [comment]
Closed for now with #40 
Possible new task for formatter improvements. (TBD with @GoebelL)
